src/helpers/blockchain_data.py

```diff
@@ -71,3 +74,34 @@ def get_auction_id(self, tx_hash: str) -> int:
         # convert bytes to int
         auction_id = int.from_bytes(call_data_bytes[-8:], byteorder="big")
         return auction_id
+
+
+def get_transaction_timestamp(tx_hash: str, web3: Web3) -> tuple[str, int]:
```

Comment:
Is there a reason these are not defined as methods of the BlockchainData class?

Reply:
It should be made a method of that class.

I just moved it around a bit and did not want to have it depend on all the other things for computing imbalances or even a chain name in this blockchain fetcher. Testing was a bit easier if those methods are standalone.

Reply:
[119658e](https://github.com/cowprotocol/token-imbalances/pull/75/commits/119658e47ce0fc5942c6fe8379fbe51e80526aa9)